### PR TITLE
Docs: Update Readme with code snippet examples for all the API methods

### PR DESCRIPTION
**Pull request by J-rayX**
This commit does the following:
- adds Snippet examples of how to use the SDK APIs.
- includes a table of content to navigate the README sections

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,11 +21,24 @@ To install this package
 ```shell
 # Via pip
 pip install novu
-
+ßß
```
Comment (ryshu) [inline README.md:24]: Hum, think it's an error

**Review by ryshu — DISMISSED** [review]
With the exception of the strange characters, everything seems OK.

File: README.md (hunk 1 above)
Comment (J-rayX) [reply README.md]: Oh, yeah @ryshu. I've made a new commit that removes the characters. Thanks.

Comment (J-rayX) [reply README.md]: > Hum, think it's an error

Oh, yeah @ryshu. I've made a new commit that removes the characters. Thanks.

**Review by ryshu — APPROVED** [review]

**Pull request merged**